dev_tools/packaging/isolated_packages_test.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
```

Comment:
I'm going to continue to strongly advocate that we start implementing some of these scripts in python.

The Matthew P. Harrigan trademarked method of determining whether to write a bash script: If you have a list, do not write a bash script :tm:

Reply:
I agree with that sentiment and I will rewrite it! Keep advocating! :)

Reply:
Fixed. Including an improved refactoring of the isolated notebook execution framework - it is now extracted out to a shared, nicely reusable pytest fixture for cloning environments without duplicated downloads of dependencies and running expensive virtualenv based tests in parallel.